Update README.md (#43)
WojciechBarczynski/BITEhack2024: README.md

# BITEhack2024 - Quit It

> Welcome to **Quit It**! - an innovative application designed to support individuals in overcoming their habits with the help of friends or aiding their loved ones in their journey to break free from addiction. With a unique feature that seamlessly integrates community support and the proximity of your closest allies, **Quit It**! provides an effective platform for both you and your friends to conquer habits together.

Reviewer: Zdania są za długie. Bardziej angażują i łatwiej się czyta krótkie.
```suggestion
Welcome to **Quit It**! - an innovative social-media platform designed to support individuals in overcoming their habits with the help of friends. Unique features that seamlessly integrate community support and the proximity of your closest provide an effective platform for both you and your friends to conquer habits together.
```

Reviewer: To też nie jest idealne. Lepiej to skrócić jeszcze

## Introduction
Sarah - a determined individual who had decided it was time to break free from the chains of her addiction. Opting not to face this journey alone, she turned to **Quit It**! an app that fostered communal support. Sarah invited her closest friends to join her on this challenging path to quit the habit.

As Sarah and her friends embarked on this collective journey, they found strength in each other's company. She knew that she had her friends on her side, allowing them to share her triumphs, struggles, and milestones. The camaraderie formed a powerful support network that [bolstered](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3917180/pdf/IJNMR-19-19.pdf) her resolve!

Unlike previous attempts to quit in isolation, Sarah noticed a significant difference this time. The support of her friends and shared milestones among them accelerated the process of overcoming her individual addictions. The determination fueled a positive and competitive spirit, turning the challenging endeavor into a shared triumph.

> **Quit It**! not only became a tool for tracking progress but also for celebrating each milestone achieved. Through the power of friendship and mutual support, Sarah discovered that quitting a habit didn't have to be a solitary struggle. With **Quit It**, she experienced firsthand how facing the journey together not only made it more manageable but also transformed it into a victory.

Reviewer: XD

Po prostu wyjaśnij dlaczego to pomaga i dodaj linki do artykułów. To jest cringe AF

Author: Sam jesteś cringe af. Naucz się robić quality review.
![review](https://github.com/WojciechBarczynski/BITEhack2024/assets/79122265/72d5779d-fdee-4406-b789-589706f258b4)

# BITEhack2024 - Quit It
Welcome to **Quit It**! - an innovative application designed to support individuals in overcoming their habits with the help of friends or aiding their loved ones in their journey to break free from addiction. With a unique feature that seamlessly integrates community support and the proximity of your closest allies, **Quit It**! provides an effective platform for both you and your friends to conquer habits together.

## Introduction
Supporting individuals in overcoming addiction plays a [crucial](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3917180/pdf/IJNMR-19-19.pdf) role in their success on the challenging journey toward health and regaining control over their lives. Being open, empathetic, and providing support can expedite the recovery process, helping individuals conquer the challenges associated with addiction. Support not only builds strong social bonds but also inspires positive changes, creating an atmosphere where every step towards health is acknowledged and encouraged.

> In the event of breaking the streak of addiction-free days, the application promptly notifies the user of this occurrence. Additionally, it provides specific information regarding potential risks associated with reverting to the addiction. For example, in the case of smoking cigarettes, the app may highlight the increased risk of developing lung cancer based on [research on Oxford](https://watermark.silverchair.com/zv800204000099.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAA1QwggNQBgkqhkiG9w0BBwagggNBMIIDPQIBADCCAzYGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM5lxTpp-hwvt6m-lYAgEQgIIDBy1IVgpXc-v6dZ6PC-7_eOKGh1dWkCDlGpS2QP8YNpjgidj2KggUs8v-5YAYOkxHrURv_hjv6_QIiFV9gn9KlxKf_f16EvU3GqMAcH2DPFlmsIUe0sMc3seNAF_ckyWqE4ELIe9tt0QUicQ2fm5iwNe0ynnbsMukD2BM16UNtc4dPjkHHnaEFAEORPaSBYcX2lHbax_PBhlUNz7EUEZtWmZ_aIwVqhAiGsTZurwZL9RuEAIxiE_mvSbm2Jc8GBpLal3GjkrRLMn0EIvUtRDqCqh4uW1pb8-lLj9skn8_cNjaXDFStpHrC141sr579gsUszPK2SnXg9_5vj9d57q4SNrCczKo_F_8wQybbnNOble_7DeWqnII8qYKa2gbD6rFI_Ail15ehx8NiXUNIaavbU1KoGwo66l6P0B3SKWoBCGBcHyNjGrizisiol3XEGNmVUuCRtt3n5ES08BRV1C9dpevw8ABoyw6UIUQNuTC9nqs0q5aRqvxLoOMTg38knv4U8SS04aTV9er3X9_xj9mCByy0xaUOmeo1-sqCCnf4FW2So5ASB0McrJ7nM4Fq-AiMA0aS07RULAAHBZLV06BX5m_L_Bpjo9x2ShvsVwchvgZk5XPXHUOuX88QXOPUOwkXPdVbjYjiF8N4khSe2Yv2ZncSm5-c0M2rZCLoOaQDMmhA8q5idC6Q9EuX_n5sm340_ZE2yDnTa6gvHtnkKM-CAxMQmYZlofzVUKJHA5h4nY--W00TYtbiCZ_Nmv0c76rUF757BQHENdRZZmjyEE94VXZndVunRjCx0NuA9EcKJGP-LmhC_L6ZOBUktXJVRvE86_hOidDU_caCkcHuKQOrTGt6S_jVpn6PQ08rNz_gL3tcE9qKaVXMlwoWqtjCu2ZSVUQiEpHme3Q24-2Bdyk2aKg4wyE0HovX_tMWcyN8kuFVRyWgLS57Sw6k73w1JhlIrSOgKMR1J2QcQLU_lbiyoqImQOEj_jFIBPwpsZSEbNiPrFuSTJtnNkUS0fh0gyhyKpIH9ICobE) and some other articles. This feature aims to emphasize the consequences of returning to the addiction and motivate the user to persist in their efforts to combat dependency.

Reviewer: ```suggestion
In the event of breaking the streak of addiction-free days, the application promptly notifies the user of this occurrence. Additionally, it provides specific information regarding potential risks associated with reverting to the addiction. For example, in the case of smoking cigarettes, the app may highlight the increased risk of developing lung cancer based on [research conducted on Oxford](https://watermark.silverchair.com/zv800204000099.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAA1QwggNQBgkqhkiG9w0BBwagggNBMIIDPQIBADCCAzYGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM5lxTpp-hwvt6m-lYAgEQgIIDBy1IVgpXc-v6dZ6PC-7_eOKGh1dWkCDlGpS2QP8YNpjgidj2KggUs8v-5YAYOkxHrURv_hjv6_QIiFV9gn9KlxKf_f16EvU3GqMAcH2DPFlmsIUe0sMc3seNAF_ckyWqE4ELIe9tt0QUicQ2fm5iwNe0ynnbsMukD2BM16UNtc4dPjkHHnaEFAEORPaSBYcX2lHbax_PBhlUNz7EUEZtWmZ_aIwVqhAiGsTZurwZL9RuEAIxiE_mvSbm2Jc8GBpLal3GjkrRLMn0EIvUtRDqCqh4uW1pb8-lLj9skn8_cNjaXDFStpHrC141sr579gsUszPK2SnXg9_5vj9d57q4SNrCczKo_F_8wQybbnNOble_7DeWqnII8qYKa2gbD6rFI_Ail15ehx8NiXUNIaavbU1KoGwo66l6P0B3SKWoBCGBcHyNjGrizisiol3XEGNmVUuCRtt3n5ES08BRV1C9dpevw8ABoyw6UIUQNuTC9nqs0q5aRqvxLoOMTg38knv4U8SS04aTV9er3X9_xj9mCByy0xaUOmeo1-sqCCnf4FW2So5ASB0McrJ7nM4Fq-AiMA0aS07RULAAHBZLV06BX5m_L_Bpjo9x2ShvsVwchvgZk5XPXHUOuX88QXOPUOwkXPdVbjYjiF8N4khSe2Yv2ZncSm5-c0M2rZCLoOaQDMmhA8q5idC6Q9EuX_n5sm340_ZE2yDnTa6gvHtnkKM-CAxMQmYZlofzVUKJHA5h4nY--W00TYtbiCZ_Nmv0c76rUF757BQHENdRZZmjyEE94VXZndVunRjCx0NuA9EcKJGP-LmhC_L6ZOBUktXJVRvE86_hOidDU_caCkcHuKQOrTGt6S_jVpn6PQ08rNz_gL3tcE9qKaVXMlwoWqtjCu2ZSVUQiEpHme3Q24-2Bdyk2aKg4wyE0HovX_tMWcyN8kuFVRyWgLS57Sw6k73w1JhlIrSOgKMR1J2QcQLU_lbiyoqImQOEj_jFIBPwpsZSEbNiPrFuSTJtnNkUS0fh0gyhyKpIH9ICobE) and some other articles. This feature aims to emphasize the consequences of returning to the addiction and motivate the user to persist in their efforts to combat dependency.
```
